src/components/loginform/LoginForm.scss

```diff
@@ -0,0 +1,189 @@
+.login-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #f2f2f2;
+  }
+  
+  .login-left {
+    flex: 1;
+    padding: 20px;
+    min-height: 100vh;
+    display: flex;
+    background: rgba(255, 255, 255, 0.8) url(../../images/loginform/Backgroundlogin.svg);
+    background-size: cover;
+  }
+  
+  .login-left img {
+    width: 100%;
+  }
+  
+  .login-right {
+    flex: 1;
+    padding: 20px;
+    border-radius: 10px;
+    display: flex;
+  flex-direction: column;
+  align-items: center;
+  }
+  
+  
+  .login-right form {
+    display: flex;
+    flex-direction: column;
+  }
+  
+  .login-right label {
+    width: 151px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    letter-spacing: -0.015em;
+    color: #292929;
+    flex: none;
+    flex-grow: 0;
+    flex-grow: 0;
+  }
+  
+  .login-right input {
+    padding: 10px;
+    border: none;
+    border-radius: 5px;
+    box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.1);
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 10px 12px 12px;
+    gap: 10px;
+    border: 1px solid #B1B5CA;
+    border-radius: 6px;
+    flex: none;
+      flex-grow: 0;
+    order: 0;
+    flex-grow: 0;
+  }
+  
+  .login-buttons {
+    display: flex;
+    flex-direction: column;
+    padding-top: 15px;
+  }
+  
+  .login-buttons button {
+    background: #1A3835;
+    border-radius: 6px;
+    color: #fff;
+    border: none;
+    padding: 10px;
+    cursor: pointer;
+  }
+  
+  .checkbox-container {
+    display: flex;
+    align-items: center;
+    margin: 10px 0;
+  }
+  
+  .checkbox-container input[type="checkbox"] {
+    margin-right: 10px;
+    cursor: pointer;
+  }
+  
+  .login-buttons a {
```

Comment:
Beautiful side of scss is that you can write this like 
```
.login-buttons{
a{

}

}
```.  

and that style will only affect a tags in .login-buttons class. And also it will be more readable